docs/deploy/helm-customization.md

```diff
@@ -0,0 +1,87 @@
+# Helm Chart Customization
+
+Local and production deployments alike can be customized by modifying the ``chart/values.yaml`` Helm chart file or a local override. For more on using local overrides, see the [Local Deployment Guide](local.md). The remainder of this guide covers some of the customization options available in the Helm chart.
+
+## Default Organization
+
+The ``default_org`` setting is used to specify the name for the default organization created in a Browsertrix Cloud deployment. A slug will be auto-generated based on this value and can be modified in Org Settings within the application.
```

Comment:
I know we haven't officially renamed it yet, but we're basically going to at this point.  Have been referring to the app as Browsertrix everywhere else recently.

Also adds a link to the user guide page.

```suggestion
The ``default_org`` setting is used to specify the name for the default organization created in a Browsertrix deployment. A slug will be auto-generated based on this value and can be modified in [Org Settings](../user-guide/org-settings.md) within the application.
```